src/types/apiInterfaces.ts

```diff
@@ -394,6 +394,7 @@ export interface IOrderInitialData {
     life_elements: Array<ILifeField>;
     flow_settings: Record<string, unknown>;
     requires_shipping: boolean;
+    rsa_enabled: boolean;
```

Comment:
I think Shagufa already added `rsa_enabled`  in the interface, it can be found at line 435

Reply:
Oh in the checkout process interface. Good catch, from the ticket I thought it'd be in the order init data interface. I'll see if I can make use of that in the express pay library.

Reply:
Yep, works good, thanks again. I'll close this PR